frontend/src/metabase/visualizations/echarts/cartesian/model/other-series.ts

```diff
@@ -83,10 +83,19 @@ export const getAggregatedOtherSeriesValue = (
   aggregationType: AggregationType = "sum",
   datum: Datum,
 ): number => {
-  const aggregation = AGGREGATION_FN_MAP[aggregationType];
   const values = seriesModels.map((model) =>
     checkNumber(datum[model.dataKey] ?? 0),
   );
+
+  return getAggregationByValues(values, aggregationType);
+};
+
+export const getAggregationByValues = (
+  values: number[],
+  aggregationType: AggregationType = "sum",
```

Comment:
it was already defaulted to sum, just keeping that around. 

Ideally if a non known type is requested then we just display an error or ignore the column?